client/templates/adminMenu.html

```diff
@@ -10,7 +10,22 @@
         <button ng-click="$mdOpenMenu()" class="admin-menu-button">
         SPEAKERS
         </button>
-        <md-menu-content>
+        <md-menu-content class="admin-menu-item">
+            <md-menu-item ng-repeat="speaker in speakers" class="md-indent admin-menu-item">
+                <md-button ng-click="removeSpeaker(speaker)" style="width: 0px" class="admin-menu-button md-no-focus">
+                    <i class="material-icons" style="margin-bottom: 15px; width: 0px">clear</i>
+                </md-button>
+                <span style="margin-top: 3px">{{speaker}}</span>
+            </md-menu-item>
+            <md-menu-divider></md-menu-divider>
+            <md-menu-item class="md-indent admin-menu-item">
+                <i class="material-icons" style="margin-bottom: 45px; width: 40px">add</i>
+                <form ng-submit="addSpeaker()">
```

Comment:
Maybe put the input at the top of the menu in case the list of speakers is long.